### PR TITLE
Chore/update main script with working placements

### PR DESCRIPTION
**Pull request by tashmulla**
**Note:** need to get the Banner ad loading properly.

### CODE REVIEW

File: demo/Assets/MainScript.cs [hunk 1]
```diff
@@ -9,7 +9,7 @@ public class MainScript : MonoBehaviour {
 
 	// Use this for initialization
 	void Start () {
-		// do nothing
+    AwesomeAds.init(true);
```
Comment (thomas-james-1986) [inline demo/Assets/MainScript.cs:12]: Is there some funky indentation going on here? 🤔

Comment (tashmulla) [reply demo/Assets/MainScript.cs:12]: Yeah I don't really get it, cos in my editor it all looks normal like this!

Screenshot 2022-04-12 at 14 07 37

Investigating what's happened 🕵️‍♀️

File: demo/Assets/MainScript.cs [hunk 2]
```diff
@@ -18,95 +18,51 @@ public class MainScript : MonoBehaviour {
...
+  // Banner
+	public void playBanner() {
+    banner = SABannerAd.createInstance();
+    banner.setCallback ((placementId, evt) => {
+      if (evt == SAEvent.adLoaded) {
+          banner.play();
+      }
+    });
+    banner.setConfigurationProduction();
+		banner.load(82088);
```
Comment (thomas-james-1986) [inline demo/Assets/MainScript.cs:34]: I wonder if these would be considered magic numbers, maybe they should be in constants?

Comment (tashmulla) [reply demo/Assets/MainScript.cs:34]: Good shout - swapping out now 🙂

File: demo/Assets/MainScript.cs (hunk 1 above)
Comment (tashmulla) [reply demo/Assets/MainScript.cs]: @thomas-james-1986 alright ran it through Prettier and I think it's sorted it now - let me know if not 🙂

Comment (thomas-james-1986) [reply demo/Assets/MainScript.cs]: Nice! Looks good

File: demo/Assets/MainScript.cs [hunk 3]
```diff
@@ -1,112 +1,77 @@
...
+
+    private const int placementIdBanner = 82088;
+
+    private const int placementIdInterstitialStandard = 82089;
+
+    private const int placementIdInterstitialKSF = 84799;
+
+    private const int placementIdVideoDirect = 82090;
+
+    private const int placementIdVPAID = 84798;
```
Comment (thomas-james-1986) [inline demo/Assets/MainScript.cs:16]: 👌

File: demo/Assets/MainScript.cs [hunk 4]
```diff
@@ -1,112 +1,77 @@
...
+    }
 
-	public void playVideo1 () {
-//		if (SAVideoAd.hasAdAvailable (32848)) {
-//			SAVideoAd.play (32848);
-//		}
-	}
+    // Direct Video
+    public void playVideo1() {
+        SAVideoAd.load(82090);
```
Comment (thomas-james-1986) [inline demo/Assets/MainScript.cs:63]: Should this be using a `const` ?

File: demo/Assets/MainScript.cs [hunk 5]
```diff
@@ -1,112 +1,77 @@
...
-//		}
-//		if (SAAppWall.hasAdAvailable (470)) {
-//			SAAppWall.play (470);
-//		} else if (SAAppWall.hasAdAvailable (437)) {
-//			SAAppWall.play (437);
-//		}
-	}
+    // VPAID via KSF
+    public void playVideo2() {
+        SAVideoAd.load(84798);
```
Comment (thomas-james-1986) [inline demo/Assets/MainScript.cs:71]: Maybe an opportunity for another `const` here?

File: demo/Assets/MainScript.cs (hunk 4 above)
Comment (tashmulla) [reply demo/Assets/MainScript.cs:63]: Oh shoot yes, good spot - updating!

**Pull request merged**